packages/ui-app/src/types.d.ts

```diff
@@ -25,3 +25,5 @@ export type BaseContext = {
     }
   }
 };
+
+export type BitLength = 32 | 64 | 128 | 256;
```

Comment:
@jacogr Should this be `8 | 16 | 32 | 64 | 128 | 256` so its the same as https://github.com/polkadot-js/api/commit/60d3d6396a8e48626c74376e5cba0bdd41f17110#diff-0dce4593d48039d48c497cf070a879cfR16

Reply:
I think (and we are going through this in the encoding atm), that all numbers are (currently) 32 (most indexes, i.e. Nonce, PropIndex, ReferendumIndex, ...) or 64 (BlockNumber, ...) or 128 (Balance). We actually don't have any values shuttled around that don't fit here.

For future-proofing, the 8 | 16 | ... is probably better. (Although it can be added as the need arrises.)

Reply:
TL;DR - We can adpat either way, but happy as it is, and won't complain if it is changed either.